modules/RockRuby.cmake

```diff
@@ -271,6 +271,16 @@ ELSEIF(NOT RUBY_EXTENSIONS_AVAILABLE)
        OUTPUT_VARIABLE RUBY_CFLAGS)
     STRING(REPLACE "\n" "" RUBY_CFLAGS ${RUBY_CFLAGS})
 
+    if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
```

Comment:
There's IMO no need to keep the CLang test here. You check if the flag exists or not, that should be enough,